types/cheerio/index.d.ts

```diff
@@ -271,6 +271,8 @@ interface CheerioAPI extends CheerioSelector, CheerioStatic {
   load(element: CheerioElement, options?: CheerioOptionsInterface): CheerioStatic;
 }
 
+interface Document { }
+
```

Comment:
Does it make sense to use this library without the dom lib?

Reply:
Yes - we're using this library in a script run via node to scrape data. Everything works just fine.